cheatsheets_draft/Webhook_Security_Guidelines_CheatSheet.md

```diff
@@ -0,0 +1,169 @@
+# Webhook Security Guidelines Cheat Sheet
+
+## Introduction
+
+Webhooks provide a mechanism where a Server-side application can notify a client-side application when a new even (that the client-side application might be interested in) has occurred on the server. Webhooks are incredibly useful and a resource-light way to implement event reactions. However, webhooks can also be abused and the following considerations must be taken into account to secure Webhooks. It should also be noted that webhook security considerations apply to both publisher/server and subscriber/client. 
+
+At the core of webhooks fundamental security best practices are available however are not enabled by default. This is a gap in the “default secure” principle however with focus and configuration this gap can be addressed. One item to note is webhook urls by default carry a high level of confidentiality. This comes from the lack of default secure principle mentioned above. With this webhook urls should be kept secret and while the security community has generally accepted that obscurity is not security it is a best practice to follow. If your webhooks are sharing sensitive information the following considerations should be reviewed and implemented.
+
+At high level, the following requirements outline the security mechanisms in various ways. These high level requirements allow engineers and developers to properly implement security controls. 
+
+
+- **Confidentiality: A passive attacker must not be able to read Webhook Notifications, as they may contain confidential data.**
+- **Integrity: An active attacker must not be able to alter Webhook Notifications, as this may result in unexpected behaviour affecting your infrastructure.**
+- **Authenticity: An active attacker must not be able to forge Webhook Notifications as this may result in your receive code taking unwanted action.**
+
+
+## Contents
+
+Controls for Securing Webhooks
+
+- Securing Webhook URLs
+-	Enable SSL/TLS
+-	Basic Authentication
+- Mutual TLS Authentication
+-	Oath Authorization
```

Comment:
[typo] Might be `Oauth`?

Reply:
OAuth :)
Not a concern right now. This is handled later on. Thanks for the note ;)